### PR TITLE
feat(GlobalAlert): adding new alert component

### PR DESCRIPTION
**Pull request by VadymBezpalko**
Resolves: #1246

## Description
New GlobalAlert component

## Storybook

https://feature-1246--613a8e945a5665003a05113b.chromatic.com

## Checklist

**Obligatory:**

- [x] Self review (use this as your final check for proposed changes before requesting the review)
- [x] Add reviewers (`livechat/design-system`)
- [x] Add correct label
- [x] Assign pull request with the correct issue

### CODE REVIEW

File: packages/react-components/src/components/GlobalAlert/GlobalAlert.module.scss [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      @include icon-color.kind();
+    }
+
+    &--expanded {
+      @include icon-color.kind($expanded: true);
+    }
+  }
+
+  &__label {
+    transition: color $animation-duration ease-out;
```
Comment (marcinsawicki) [inline packages/react-components/src/components/GlobalAlert/GlobalAlert.module.scss:61]: Shouldn't we use token here? 🤔

File: packages/react-components/src/components/GlobalAlert/GlobalAlert.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { vi } from 'vitest';
+
+import { GlobalAlert, IGlobalAlertProps } from './GlobalAlert';
+
+describe('GlobalAlert', () => {
```
Comment (marcinsawicki) [inline packages/react-components/src/components/GlobalAlert/GlobalAlert.spec.tsx:9]:
```suggestion
describe('<GlobalAlert> component', () => {
```

**Review by marcinsawicki — APPROVED** [review]

**Comment by vladko-uxds** [comment]
update media query from 1360 to 1430px
image

**Review by vladko-uxds — APPROVED** [review]
👍

**Comment by VadymBezpalko** [comment]
I'm closing this PR because Alert was redesigned in meantime and the new Alert version is available here: https://github.com/livechat/design-system/pull/1277

**Pull request closed without merging**